AcceptanceTests/AcceptanceTests.swift

```diff
@@ -193,65 +192,75 @@ extension AcceptanceTests {
     var muterXcodeOutput: String {
         return contentsOfFileAsString(at: muterXcodeOutputPath)
     }
-
+    
     var muterEmptyStateOutputPath: String { return "\(AcceptanceTests().rootTestDirectory)/muters_empty_state_output.txt" }
     var muterEmptyStateOutput: String {
         return contentsOfFileAsString(at: muterEmptyStateOutputPath)
     }
-
+    
     var muterAbortedTestingOutputPath: String { return "\(AcceptanceTests().rootTestDirectory)/muters_aborted_testing_output.txt" }
     var muterAbortedTestingOutput: String {
         return contentsOfFileAsString(at: muterAbortedTestingOutputPath)
     }
     
+    var muterHelpOutputPath: String { return "\(AcceptanceTests().rootTestDirectory)/muters_help_output.txt" }
+    var muterHelpOutput: String {
+        return contentsOfFileAsString(at: muterHelpOutputPath)
+    }
+    
     var muterLogsRootPath: String { return "\(AcceptanceTests().rootTestDirectory)/muter_logs/" }
     var muterLogFiles: [String] {
         return contentsOfDirectory(at: muterLogsRootPath)
             .map { muterLogsRootPath + "/" + $0 }
-            .flatMap (contentsOfDirectory(at:))
+            .flatMap(contentsOfDirectory(at:))
     }
     
     var createdIOSConfigurationPath: String { return "\(AcceptanceTests().rootTestDirectory)/created_iOS_config.json" }
     var createdIOSConfiguration: Data {
         return contentsOfFileAsData(at: createdIOSConfigurationPath)
     }
-
+    
     var createdMacOSConfigurationPath: String { return "\(AcceptanceTests().rootTestDirectory)/created_macOS_config.json" }
     var createdMacOSConfiguration: Data {
         return contentsOfFileAsData(at: createdMacOSConfigurationPath)
     }
+}
+
+@available(OSX 10.13, *)
+extension AcceptanceTests {
     
-    var muterHelpOutputPath: String { return "\(AcceptanceTests().rootTestDirectory)/muters_help_output.txt" }
-    var muterHelpOutput: String {
-        return contentsOfFileAsString(at: muterHelpOutputPath)
+    func contentsOfLogFile(named fileName: String) -> String {
+        return contentsOfDirectory(at: muterLogsRootPath)
+            .first
+            .map { muterLogsRootPath + $0 + "/" + fileName }
+            .map (contentsOfFileAsString(at:))!
     }
-
+    
+    func contentsOfDirectory(at path: String) -> [String] {
+        return try! FileManager
+            .default
+            .contentsOfDirectory(atPath: path)
+            .exclude { $0.starts(with: ".") } // this filters out hidden files/folders
+    }
+    
     func contentsOfFileAsString(at path: String) -> String {
         guard let data = FileManager.default.contents(atPath: path),
             let output = String(data: data, encoding: .utf8) else {
-                fatalError("Unable to find a valid output file from a prior run of Muter at \(path)")
+                fatalError("u dun fuked up, son.")
```

Comment:
HAHAHAHAHAHA
that's funny but I think we should use another wording?

`Unable to load file at path: \(path)`